views/Main.render.react.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import {render} from 'react-dom';
+import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 
 import MainPage from './Main.react';
 import store from '../stores/store';
 
 const reduxStore = store();
 
-render(<Provider store={reduxStore}><MainPage/></Provider>, document.getElementById('main'));
+render(<Provider store={reduxStore}><MainPage /></Provider>, document.getElementById('main'));
```

Comment:
Just a question, did you do this white space adjustment through https://codebeautify.org/ like tool or manually?

Reply:
Through a tool, I used the [Prettier VSCode extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).